src/modules/commander/commander_params.c

```diff
@@ -1059,3 +1059,95 @@ PARAM_DEFINE_FLOAT(COM_THROW_SPEED, 5);
  * @increment 1
  */
 PARAM_DEFINE_INT32(COM_FLTT_LOW_ACT, 3);
+
+/**
+ * It is the color that indicates that the mission is suitable.
+ *
+ * @group Commander
+ * @value 0 COLOR_OFF
+ * @value 1 COLOR_RED
+ * @value 2 COLOR_GREEN
+ * @value 3 COLOR_BLUE
+ * @value 4 COLOR_YELLOW
+ * @value 5 COLOR_PURPLE
+ * @value 6 COLOR_AMBER
+ * @value 7 COLOR_CYAN
+ * @value 8 COLOR_WHITE
+ */
+PARAM_DEFINE_INT32(COM_MISN_O_CLR, 2);
+
+/**
+ * It is the color that indicates that the mission is warning.
+ *
+ * @group Commander
+ * @value 0 COLOR_OFF
+ * @value 1 COLOR_RED
+ * @value 2 COLOR_GREEN
+ * @value 3 COLOR_BLUE
+ * @value 4 COLOR_YELLOW
+ * @value 5 COLOR_PURPLE
+ * @value 6 COLOR_AMBER
+ * @value 7 COLOR_CYAN
+ * @value 8 COLOR_WHITE
+ */
+PARAM_DEFINE_INT32(COM_MISN_W_CLR, 4);
+/**
+ * It is the color that indicates that the mission is fail.
+ *
+ * @group Commander
+ * @value 0 COLOR_OFF
+ * @value 1 COLOR_RED
+ * @value 2 COLOR_GREEN
+ * @value 3 COLOR_BLUE
+ * @value 4 COLOR_YELLOW
+ * @value 5 COLOR_PURPLE
+ * @value 6 COLOR_AMBER
+ * @value 7 COLOR_CYAN
+ * @value 8 COLOR_WHITE
+ */
+PARAM_DEFINE_INT32(COM_MISN_F_CLR, 1);
+
+/**
+ * It is the mode that indicates that the mission is suitable.
+ *
+ * @group Commander
+ * @value 0 MODE_OFF # turn LED off
```

Comment:
I would change the wording MODE_OFF, MODE_ON, etc is what the user is going to see in QGroundControl

```suggestion
 * @value 0 Off
```